src/core/components/button/button.test.tsx

```diff
@@ -66,31 +67,34 @@ describe('<Button /> component', () => {
         expect(link.href).toEqual(href);
     });
 
-    it('disables the button on disabled state', () => {
+    it('disables the button on disabled state', async () => {
+        const user = userEvent.setup();
```

Comment:
I just saw that testing-library recommends always using the `.setup` function before using their functionality, good point 👌🏻 Just one comment though, it would be nice if the `setup` usage would be aligned across all tests. Do you think we can update the other tests using the `userEvent` library to call the setup function on this PR or does it affects too many files?

Reply:
Sured, updated here: https://github.com/aragon/ods/pull/213/commits/1ebf9e47596588f0861262fcae2273b34b93004b